modules/administration/pages/troubleshooting/tshoot-autoinstallation.adoc

```diff
@@ -1,17 +1,11 @@
 [[troubleshooting-autoinstallation]]
 = Troubleshooting Autoinstallation
 
-Depending on your base channel, new autoinstallation profiles might be subscribed to a channel that is missing required packages.
+If you use the obsolete plain {salt} (salt-minion) implementation for communication, you on your own must make sure that all the needed software (software packages) is properly available in configured client channels.
```

Comment:
I am not sure about the work "obsolete" . Only together with SUSE Manager we do not use this package anymore. But for SLES it is still a regular supported package. But I am unsure how to phrase this correctly

Reply:
ok, thanks for bg info.  I tried to change it that way.